ufpy/ustack.py

```diff
@@ -0,0 +1,48 @@
+# -*- encodibg: utf-8 -*-
+from __future__ import annotations
+from typing import Any
+
+class UStack():
+    def __init__(self, __list: list | tuple = list()) -> UStack:
+        """
+        UStack's methods:
+            append(**items) - to add item to begin.
+            empty() - true if stack is empty.
+            pop() - delete the last element in stack.
+        """
+        self.__list = __list
+        
+    def pop(self) -> Any:
+        return self.__list.pop()
+
+    @property
+    def top(self) -> Any | bool:
+        if self:
+            return self.__list[-1]
+        else:
+            return False
+    
+    def top(self) -> Any | bool:
+        if self:
+            return self.__list[-1]
+        else:
+            return False
+
+    @property
+    def empty(self) -> bool:
```

Comment:
Not i'm not asking you to create from empty property. I'd asking that you should rename empty() to __nonzero__.
You can rename empty() method to is_empty() and create new __nonzero__ magic method

```py
def is_empty(self) -> bool:
    return len(self.__list) == 0

def __nonzero__(self) -> bool:
    return not self.is_empty()
```

Reply:
also you can use `len(self)` instead of `len(self.__list)`

Reply:
Okay, i will do it tomorrow. Can you speak russian?

Reply:
> Okay, i will do it tomorrow. Can you speak russian?

yes, but in this repo we can't speak any language except English for other contributors)